stdlib/src/builtin/string.mojo

```diff
@@ -438,6 +438,17 @@ struct String(
     var _buffer: Self._buffer_type
     """The underlying storage for the string."""
 
+    """ Usefull string aliases. """
+    alias WHITESPACE = String(" \n\t\r\f\v")
+    alias ASCII_LOWERCASE = String("abcdefghijklmnopqrstuvwxyz")
+    alias ASCII_UPPERCASE = String("ABCDEFGHIJKLMNOPQRSTUVWXYZ")
+    alias ASCII_LETTERS = String.ASCII_LOWERCASE + String.ASCII_UPPERCASE
+    alias DIGITS = String("0123456789")
+    alias HEXDIGITS = String.DIGITS + String("abcdef") + String("ABCDEF")
+    alias OCTDIGITS = String("01234567")
+    alias PUNCTUATION = String("""!"#$%&'()*+,-./:;<=>?@[\\]^_`{|}~""")
+    alias PRINTABLE = String.DIGITS + String.ASCII_LETTERS + String.PUNCTUATION + String.WHITESPACE
```

Comment:
A note for the maintainers: I don't have an opinion here, but I'm just pointing out that in Python, those constants are lowercase: https://docs.python.org/3/library/string.html#string-constants . We just need to be aware of this fact before merging.

Reply:
I would prefer to keep them uppercase. But this is question to Joe Loser I think